### PR TITLE
(Feature) Select input for Boolean type for contract type acc

### PR DESCRIPTION
**Pull request by vbaranov**
screen shot 2018-12-19 at 19 52 25

Based on https://github.com/poanetwork/nifty-wallet/pull/219

### CODE REVIEW

**Comment by dennis00010011b** [comment]
@vbaranov 
Execute method: parameter field doesn't allow input for functions with no boolean return
screen shot 2018-12-19 at 09 31 35

**Comment by vbaranov** [comment]
>Execute method: parameter field doesn't allow input for functions with no boolean return

Thanks, @dennis00010011b. Fixed here https://github.com/poanetwork/nifty-wallet/pull/229/commits/2f89dfc5bbf8f9a1fe3816ce78713faba76368a7

File: old-ui/app/components/send/send-contract.js [hunk 1]
```diff
@@ -174,16 +213,28 @@ class SendTransactionScreen extends PersistentForm {
...
-				placeholder={params.type}
-				defaultValue={defaultValue}
-				onChange={e => isInput ? this.handleInputChange(e.target.value, params.type, ind) : null}
-			/>
-		)
+		let field
+		if (params.type === 'bool' && isInput) {
+			field = (
+				<SendTransactionInputSelect
+					key={Math.random()}
```
Comment (fvictorio) [inline old-ui/app/components/send/send-contract.js]: Why is this necesssary?

Comment (vbaranov) [reply old-ui/app/components/send/send-contract.js]: Indeed, `key` prop is not necessary for inputs. Removed, thanks @fvictorio

**Pull request merged**